tests/Jenkins/dirac_ci.sh

```diff
@@ -127,7 +127,7 @@ installSite() {
   bash "installer.sh"
   rm "installer.sh"
   echo "source \"$PWD/diracos/diracosrc\"" > "$PWD/bashrc"
-  # TODO: This will be fixed properly as part of https://github.com/DIRACGrid/DIRAC/issues/5082
+    # TODO: This will be fixed properly as part of https://github.com/DIRACGrid/DIRAC/issues/5082
   mv "${SERVERINSTALLDIR}/etc/grid-security/"* "${SERVERINSTALLDIR}/diracos/etc/grid-security/"
   rm -rf "${SERVERINSTALLDIR}/etc"
```

Comment:
```suggestion
  # TODO: This will be fixed properly as part of https://github.com/DIRACGrid/DIRAC/issues/5082
  mv "${SERVERINSTALLDIR}/etc/grid-security/"* "${SERVERINSTALLDIR}/diracos/etc/grid-security/"
  rm -rf "${SERVERINSTALLDIR}/etc"
```

Reply:
BTW https://github.com/DIRACGrid/DIRAC/issues/5082 is closed but the issue above is still there. I can simply remove the comment...

Reply:
For the integration tests we're not using a versioned install so I agree the comment should be removed. (I thought about changing this but I don't think it's worth it.)